content/news/this-week-in-fluvio-0051.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "This Week in Fluvio #51"
+date: 2023-04-11
+weight: 20
+---
+Fluvio is a distributed, programmable streaming platform written in Rust.
+
+---
+Welcome to the 51st edition of this week in Fluvio.
+
+For this edition we have more details on **Community Contribution** 
+
+## Community update
+[Carson Rajcan](https://github.com/crajcan) presented to our team a really cool contribution to the Fluvio OSS.
+
+In the project Carson developed the functionality to apply Smart Module transformation for Producer - [PR #3014](crecdfhttps://github.com/infinyon/fluvio/pull/3014)
+
+Here is Carson's experience...
+
+## The Problem
```

Comment:
We should add a bit of context on why SmartModules are useful on the producer side. 
"Shaping  the data before it enters the topic, it spares consuming services from this burden." For example, an upstream devices does not conform new data formats, or you prefer to convert Avro to JSON for downstream processing.

Reply:
I incorporated but kept it simple to keep close to Carson's voice